client/packages/theia-integration/src/browser/diagram/glsp-theia-sprotty-connector.ts

```diff
@@ -60,8 +63,18 @@ export class GLSPTheiaSprottyConnector implements TheiaSprottyConnector, GLSPThe
 
     showStatus(widgetId: string, status: ServerStatusAction): void {
         const widget = this.widgetManager.getWidgets(this.diagramManager.id).find(w => w.id === widgetId);
+
         if (widget instanceof DiagramWidget)
             widget.setStatus(status);
+
+        if (status.severity === "ERROR") {
+            const details = isGLSPServerStatusAction(status) ? status.details : "";
+            this.messageService.error(status.message, "Show details").then(result => {
```

Comment:
Not sure if this is possible but sometimes the error does not provide any details. In that case maybe the popup should not show the 'Show details' at all as we end up with an empty message box:

![Selection_087](https://user-images.githubusercontent.com/19170971/62539714-2d680200-b856-11e9-8924-ad61a57dd388.png)
![Selection_088](https://user-images.githubusercontent.com/19170971/62539721-3062f280-b856-11e9-8f7f-8a8c6bd8c052.png)